ids/id.go

```diff
@@ -32,11 +32,39 @@ var (
 // ID wraps a 32 byte hash used as an identifier
 type ID [IDLen]byte
 
+// Writeable is an helper which helps modifiying ID content
+func Writable(id *ID) []byte {
+	return id[:]
+}
+
+// Any modification to Bytes will be lost since id is passed-by-value
+// Use Writable(ID) is you need to modify the ID
+func (id ID) Bytes() []byte {
+	return id[:]
+}
+
 // ToID attempt to convert a byte slice into an id
 func ToID(bytes []byte) (ID, error) {
 	return hashing.ToHash256(bytes)
 }
 
+// UnsafeToID attempt to convert a byte slice into an id
+// It's up to caller to make sure bytes are well formatted.
+// Should be used in tests only.
+func UnsafeToID(bytes []byte) ID {
+	rightLenSlice := make([]byte, IDLen)
+	copy(rightLenSlice, bytes)
+	res, _ := hashing.ToHash256(rightLenSlice)
+	return res
+}
```

Comment:
1. I personally prefer just keeping the error and just ignoring it in tests if we don't care about it
2. If we do need this, can we just call `ToID` and ignore the error?